package.json

```diff
@@ -4,8 +4,10 @@
   "description": "React-Relay-GraphQL Starter Kit",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "npm run start:windows",
+    "start:linux": "nodemon --exec babel-node server --presets es2015 & webpack --watch --config internals/webpack.config.js",
+    "start:windows": "start nodemon --exec babel-node server --presets es2015 & start webpack --watch --config internals/webpack.config.js",
```

Comment:
Can we rename internals to something like config, or scripts, or server? I think internals is a little confusing